lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -1002,7 +1013,7 @@ feeEstimationRegressionSpec = describe "Regression tests" $ do
                     { requiredCost = Ledger.Coin $ intCast requiredCostLovelace
                     , shortfall = Ledger.Coin 100_000
                     }
-        result <- runExceptT (calculateFeePercentiles estimateFee)
+        result <- runExceptT (calculateFeePercentiles @_ @era estimateFee)
```

Comment:
Not specific to this PR but just thinking about our direction: Maybe the need for additional type-application is a bad sign 🤔